### PR TITLE
Add argocd-staging namespace to the app-of-app-sets

### PR DESCRIPTION
**Pull request by Roming22**
Fix an error when running `hack/bootstrap-cluster.sh`. The application could not be created because the target namespace did not exist.

### CODE REVIEW

**Comment by Roming22** [comment]
@Michkov I'm surprised I ran into this issue. Is it user error?

**Comment by Michkov** [comment]
@Roming22 it failed to install toolchain-member operator which is being installed before applying all-apps ArgoCD. But also gather logs failed which maybe points to some issue with cluster.

File: argo-cd-apps/app-of-app-sets/base/kustomization.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
+  - namespace.yaml
```
[the remark this answers is not in the document]
Comment (Michkov) [reply argo-cd-apps/app-of-app-sets/base/kustomization.yaml:4]: even it's deployed to different cluster which is hosting ArgoCD.
CC @gbenhaim

[the remark this answers is not in the document]
Comment (Roming22) [reply argo-cd-apps/app-of-app-sets/base/kustomization.yaml:4]: It ended in error for me, which is why i submitted the PR.

**Pull request closed without merging**